src/engine/enginebuffer.cpp

```diff
@@ -789,12 +788,17 @@ void EngineBuffer::process(CSAMPLE* pOutput, const int iBufferSize) {
 
         // TODO(owen): Maybe change this so that rubberband doesn't disable
         // keylock on scratch. (just check m_pScaleKeylock == m_pScaleST)
-        if (is_scratching || fabs(speed) > 1.9) {
-            // Scratching and high speeds with Soundtouch always disables keylock
+        if (is_scratching || fabs(speed) > 1.9 || fabs(speed) < 0.1) {
+            // Scratching and high or low speeds with Soundtouch always disables keylock
```

Comment:
Switching to Soundtouch will produce a minimal seek when switching to it. 
I am afraid that will be bearable at such very slow speeds. Please apply the tweak only for Rubberband which does not suffer the issue.